client/src/main/java/gui/controllers/PlacementSceneController.java

```diff
@@ -61,9 +60,13 @@ public void initialize() {
 
     @FXML
     private void placeRandom() {
+        appInstance.getServerConnection().sendMessage("ASK_FOR_FLEET");
+
+        String message = appInstance.getServerConnection().getMessage();
+
         printingBoard.getChildren().removeIf(node -> node instanceof Shape);
```

Comment:
Shape is a subclass of Node, so expression "node instanceof Shape" is ambiguous.
I suggest "element -> element instanceof Shape"